ui/analyse/src/interfaces.ts

```diff
@@ -53,6 +53,9 @@ export interface ServerEvalData {
   analysis?: Analysis;
   tree: Tree.Node;
   division?: Division;
+  node: Tree.Node;
+  path: Tree.Path;
+  dests: string
```

Comment:
no these are not from server evals. I think you're confusing server eval and local eval (aka ceval)

Reply:
I might still be confused, but I think the updated PR is correct. I am unsure what to name interface "D" or "L".